srcpkgs/breeze/template

```diff
@@ -20,3 +20,10 @@ if [ "$CROSS_BUILD" ]; then
 	hostmakedepends+=" kpackage-devel kconfig-devel kcoreaddons-devel plasma-framework"
 	configure_args+=" -DKF5_HOST_TOOLING=/usr/lib/cmake"
 fi
+
+breeze-snow-cursor-theme_package() {
+	short_desc+=" - Breeze Snow cursor theme"
```

Comment:
Too long, just short_desc="Breeze Snow cursor theme" maybe?

Reply:
works for me